cli/smartcontract/smart_contract.go

```diff
@@ -130,27 +130,27 @@ func NewCommands() []cli.Command {
 				Flags: []cli.Flag{
 					cli.StringFlag{
 						Name:  "in, i",
-						Usage: "Input file for the smart contract to be compiled",
+						Usage: "Input file for the smart contract to be compiled (*.go file or directory). Omit all output-related flags to autocomplete",
 					},
 					cli.StringFlag{
 						Name:  "out, o",
-						Usage: "Output of the compiled contract",
+						Usage: "Output of the compiled contract (omit all output flags for autocompletion)",
```

Comment:
"autocompletion" sounds too magic. It should be like "if not specified _this_ is what you get"

Reply:
Fixed.